test/lit.cfg

```diff
@@ -1587,6 +1583,7 @@ config.substitutions.append(('%xcode-extra-frameworks-dir', extra_frameworks_dir
 
 config.substitutions.append(('%target-swiftmodule-name', run_cpu + '.swiftmodule'))
 config.substitutions.append(('%target-swiftdoc-name', run_cpu + '.swiftdoc'))
+config.substitutions.append(('%target-swiftinterface-name', run_cpu + '.swiftinterface'))
```

Comment:
Boo. I was really trying to avoid having another of these. Let's just be explicit in tests now that it works: `%target-cpu.swiftinterface`.

Reply:
That was my first implementation, but I changed to this to match the others.

I will change it back.